core/rfb.js

```diff
@@ -961,7 +961,7 @@ export default class RFB extends EventTargetMixin {
     _negotiate_std_vnc_auth() {
         if (this._sock.rQwait("auth challenge", 16)) { return false; }
 
-        if (!this._rfb_credentials.password) {
+        if (this._rfb_credentials.password === undefined) {
```

Comment:
Could you add the same check for `undefined` for username, password, and target in the function `_negotiate_xvp_auth()` above?

Reply:
Sure